dspace-server-webapp/src/main/java/org/dspace/app/rest/security/PoolTaskRestPermissionEvaluatorPlugin.java

```diff
@@ -51,7 +51,7 @@
     public boolean hasDSpacePermission(Authentication authentication, Serializable targetId,
                                  String targetType, DSpaceRestPermission permission) {
 
-        if (Constants.getTypeID(targetType) != Constants.POOLTASK) {
+        if (!StringUtils.equals("POOLTASK", targetType)) {
```

Comment:
Same here, this should still be a constant somewhere (need not be in the actual `Constants` class though)

Reply:
Done.